fili-sql/src/main/java/com/yahoo/bard/webservice/sql/evaluator/PostAggregationEvaluator.java

```diff
@@ -158,8 +185,14 @@ public RexNode evaluate(
             case DIVIDE:
                 List<RexNode> temp = new ArrayList<>();
                 //cast Integer to Double to avoid truncation
-                temp.add(builder.call(SqlStdOperatorTable.MULTIPLY, builder.literal(1.0), innerFields.get(0)));
-                temp.addAll(innerFields.subList(1, innerFields.size()));
+                RexNode numerator =
+                        builder.call(SqlStdOperatorTable.MULTIPLY, builder.literal(1.0), innerFields.get(0));
+                RexNode denominator = innerFields.get(1);
+                if (denominator.getKind() == SqlKind.AS) {
+                    denominator = ((RexCall) denominator).operands.get(0);
+                }
+                temp.add(numerator);
+                temp.add(denominator);
```

Comment:
Might want to add an error if none of these cases are hit.

Reply:
Based on Kevin's suggestion added an `assert innerFields.size() == 2;` so it should always hit the cases now.